utils.js

```diff
@@ -690,9 +698,9 @@ function formatCookie(arr, url) {
 
 function formatThread(data) {
   return {
-    threadID: data.thread_fbid.toString(),
-    participants: data.participants.map(function(v) { return v.replace('fbid:', ''); }),
-    participantIDs: data.participants.map(function(v) { return v.replace('fbid:', ''); }),
+    threadID: formatID(data.thread_fbid.toString()),
+    participants: data.participants.map(function(v) { return formatID(v.replace('fbid:', '')); }),
```

Comment:
Oh wow I really shouldn't have missed that.. I should stop just mindlessly copy pasting lol